syftlib/src/test/java/org/openmined/syft/SyftTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.openmined.syft
+
+import com.nhaarman.mockitokotlin2.mock
+import org.junit.jupiter.api.Test
+import org.openmined.syft.threading.ProcessSchedulers
+
+
+internal class SyftTest {
+
+    @Test
+    fun `Given socket protocol is not wss then constructor will return a null object`() {
+        val workerId = "workerId"
+        val keepAliveTimeout = 20000
+        val url = "ws://someAddress"
```

Comment:
`ws => wss`

Reply:
The intention of this test is to make it fail